### ISSUE TITLE
Virtua Tennis - ball texture is broken when RTT buffer is enabled

### ISSUE THREAD

**Issue by mlnlbrt** [libretro/flycast#95]
RTT buffer trashes ball texture in Virtua Tennis:
![42103603-dca0297a-7bc9-11e8-9b9f-fa537277645c](https://user-images.githubusercontent.com/11428910/42136152-b0358ea0-7d56-11e8-826d-f4a88765c47b.png)

**Comment by inactive123** [comment]
With OIT renderer, or non-OIT renderer, or both?

**Comment by mlnlbrt** [comment]
@twinaphex both

**Comment by inactive123** [comment]
Alright, paging @flyinghead. Seems this issue keeps haunting us a bit.

**Comment by flyinghead** [comment]
Yeah, there is still a memory corruption problem with RTT but only with RTT to buffer enabled.
Any reason why you have it enabled? It shouldn't be needed for this game.

**Comment by mlnlbrt** [comment]
@flyinghead just had it enabled for no reason
